core-api/src/main/java/com/optimizely/ab/config/Experiment.java

```diff
@@ -44,7 +44,7 @@
     private final String groupId;
 
     private final List<String> audienceIds;
-    private final Condition<AudienceIdCondition> audienceConditions;
+    private final Condition audienceConditions;
```

Comment:
Not sure about implication of this change. Is it required?

Reply:
Not required, Was testing something and forgot to put back. Will change back, thanks for catching!